src/core/importType.js

```diff
@@ -1,6 +1,8 @@
+import * as PATH from 'path'
```

Comment:
```suggestion
import path from 'path'
```
`import * as` is only for metaprogramming or laziness.

Reply:
Done. Was not aware Babel is differentiating es6 default export and commonjs module export here.
Classically I use webpack for this resolving (babel is not transpiling the imports) and webpack doesn't do this distinction.

PATH is preserved in uppercase to avoid collision will all **path** local var in the code.